src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -2812,8 +2813,10 @@ public String getPositivePrefix () {
      * <P>Examples: +123, $123, sFr123
      *
      * @param newValue the new positive prefix
```

Comment:
Although it may be apparent with the `@throws` tag, `@param` can also mention the new value is non-null.